configs/celestia/rollup.toml

```diff
@@ -17,6 +17,14 @@ signer_addres = "celestia1a68m2l85zn5xh0l07clk4rfvnezhywc53g8x7s"
 # The path to the rollup's data directory. Paths that do not begin with `/` are interpreted as relative paths.
 path = "./test-data/rollup-starter-data-celestia"
 
+user_commit_concurrency = 6
+# This is enough for getting started, but production environment needs more
```

Comment:
Can we explain how to calculate the correct amount or link to the relevant docs?